proposals/p2006.md

```diff
@@ -695,6 +696,30 @@ Carbon unsurprising in the world of programming languages which is a good thing.
 Using `var` to help signify the allocation of storage and given it also having
 widespread usage across popular languages.
 
+### Make `const` a postfix rather than prefix operator
+
+Using a prefix operator for `const` and a postfix operator for `*` causes them
+to require parentheses for complex nesting. We could avoid that by using a
+postfix `const` and this would also allow more combinations in Carbon to be
+valid in C++ with the same meaning as Carbon such as `T* const`.
+
+This direction isn't pursued because:
+
+-   We expect pointers-to-`const` to be significantly more common in code than
+    `const`-pointers-to-non-`const`, even more-so than is already the case in
+    C++. And for `pointers-to-const`, we lean towards matching more widespread
+    convention of `const T*` rather than `T const*`.
+    -   We are aware that some developers have a stylistic preference for "East
+        `const`" which would write `T const*` in C++. However, that preference
+        and advocacy for this style hasn't yet caused it to become more
+        widespread or widely adopted style.
```

Comment:
[No action required.] We could even allow both styles, by permitting `const` as either a prefix or postfix operator. This would match C++ better, including weird C++ constructs like `const T* const` (which could also be written as `T const* const` or `const (const T*)` or even `const (T const*)`).

Reply:
=] That's ... something.

I'm landing this without any action as it's not clear this is worth reviewing more words for this proposal.